src/components/POLine/Item/ItemForm.js

```diff
@@ -174,6 +178,17 @@ class ItemForm extends Component {
               name="details.receivingNote"
             />
           </Col>
+          <Col xs={6}>
+            <Field
```

Comment:
don't forget that we have templates now and they shouldn't be broken with it

Reply:
@NikitaSedyx do you mean to check if templates work with this change?

Reply:
yep